popSimManu.tex

```diff
@@ -389,58 +401,54 @@ \subsection*{The species catalog}
-\citep{kelleher2016efficient,kelleher2018efficient}.
-The tree sequence format could also be converted
-to other formats (e.g., VCF) by the user if desired.
 Currently, \stdpopsim uses the  \texttt{msprime} coalescent simulator \citep{kelleher2016efficient}
 as the default simulation engine.
-
 Coalescent simulations, while highly efficient, are limited in their ability to model
-complex selection scenarios, such as recurrent sweeps and background selection. For these reasons,
+continuous geography or 
+complex selection scenarios, such as recurrent sweeps and background selection.
+For these reasons,
 we have also implemented the forward-time simulator, \texttt{SLiM} \citep{haller2019tree,haller2019slim},
 as an alternative backend engine to \texttt{stdpopsim}, allowing for the simulation of processes that cannot be modeled under
 the coalescent. However, as forward-time simulators explicitly model all individuals in a population, simulating large
 population sizes can be highly demanding of computational resources. One common practice used to address this challenge is to
-scale down the population size used in the simulations (\textit{N}),
-scale down times specified in the demographic model by the same factor, and scale up the mutation and recombination
-rates and selection coefficients such that the scaled parameters, \textit{N$\mu$}, \textit{Nr},
-and \textit{Ns}, remain unchanged \citep[see the \texttt{SLiM} manual and][]{uricchio2014robust}.
+simulate a \emph{smaller} population, but to rescale resulting times,
+mutation rates, recombination rates, and selection coefficients
+so that the intensity of mutation, recombination, and allele frequency change due to selection
+per unit time remains the same
+%% PLR: the following is not worded quite right, or at least confusingly, because if you scale time
+%% then this scales rates also; I've opted for a less precise but more descriptive formulation?
+% scale down the population size used in the simulations (\textit{N}),
+% scale down times specified in the demographic model by the same factor, and scale up the mutation and recombination
+% rates and selection coefficients such that the scaled parameters, \textit{N$\mu$}, \textit{Nr}, and \textit{Ns}, remain unchanged
+\citep[see the \texttt{SLiM} manual and][]{uricchio2014robust}.
+Our implementation of the \texttt{SLiM} backend allows easy use of this \emph{rescaling}
+through a single ``scaling factor'' argument.
 Such down-scaled simulations are not completely equivalent to simulating
-all individuals in the population, and may lead to subtle differences.
-However, since many sequence-based measures of population diversity remain unchanged when rescaling in this fashion,
+all individuals in the population, and may lead to subtle differences,
+especially in the presence of selection.
+However, since many sequence-based measures of population diversity remain nearly unchanged when rescaling in this fashion,
```

Comment:
"nearly" because we know they're not *exactly* the same, due to differences in continuous/discrete time for instance.